### PR TITLE
chore: minor typos in the docs

### PR DESCRIPTION
**Pull request by tudorpintea999**
## Description

Fixes minor typos in the docs

Hope it helps,
Br,
Tudor.

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None, docs only

## Testing

None

## Screenshots (if applicable)

### CODE REVIEW

File: docs/file-structure.md [hunk 1]
```diff
@@ -36,9 +36,9 @@ Another common thing that makes components bloated is defining multiple function
 2. If anyone decides that they can reuse that code it's a quick and easy move to a shared folder.
 3. It can be confusing when you are looking for a component that is exported from a file that does not share the same name. It sounds trivial but it removes just a little bit of friction.
 
-### Use your judgement when splitting out components
+### Use your judgment when splitting out components
```
Comment (gomesalexandre) [inline docs/file-structure.md:39]: Sounds reasonable to me - while both spellings are technically valid, `judgement` is the old British English and now rarely used spelling.

**Review by gomesalexandre — APPROVED** [review]
Thank you for the contribution @tudorpintea999 !
Note we conventionally use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for both commits and PRs titles.

I've also modified the PR description to match our template, please make sure to keep it untouched - absolutely fine to leave blanks if not applicable.

**Comment by tudorpintea999** [comment]
hello, gotcha, i ll use that for future prs 🤝

File: docs/file-structure.md (hunk 1 above)
Comment (0xApotheosis) [reply docs/file-structure.md:39]: Unless you're an Aussie! 😁

**Pull request merged**